### ISSUE TITLE
Crash at boot with MysticalAgriculture-1.12.2-1.7.5

### ISSUE THREAD

**Issue by LemADEC** [BlakeBr0/MysticalAgradditions#45]
As of 
Cucumber-1.12.2-1.1.3
Mantle-1.12-1.3.3.49
MysticalAgriculture-1.12.2-1.7.5
TConstruct-1.12.2-2.12.0.135
MysticalAgradditions-1.12.2-1.3.1
game will crash at boot:
```
java.lang.NoClassDefFoundError: com/blakebr0/mysticalagriculture/util/Utils
	at com.blakebr0.mysticalagradditions.tinkers.TraitProsperous.getLocalizedName(TraitProsperous.java:24)
```

 https://gist.github.com/LemADEC/ef7b015388f151f205ea7c3d8354380e

Looks like MysticalAgriculture updated its structure, breaking compatibility with MysticalAgradditions .

**Comment by NieLeben** [comment]
I have the same problem. It seems like the Utils class moved from MysticalAgriculture to [Cucumber](https://github.com/BlakeBr0/Cucumber/blob/master/src/main/java/com/blakebr0/cucumber/util/Utils.java).

**Comment by BlakeBr0** [comment]
Should be fixed in version 1.3.2 of agradditions
